packages/calling-component-bindings/review/beta/calling-component-bindings.api.md

```diff
@@ -195,6 +196,9 @@ export type ParticipantsButtonSelector = (state: CallClientState, props: Calling
     myUserId: string;
 };
 
+// @public
+export type RemoveParticipantHandler = ((userId: string) => Promise<void>) & ((participant: CommunicationIdentifier) => Promise<void>);
```

Comment:
Why this is an & between 2 different functions?

Reply:
This is the syntax for function overload when a function is defined as a type